src/main/java/chess/Board.java

```diff
@@ -0,0 +1,27 @@
+package chess;
+
+import pieces.Pawn;
+
+import java.util.ArrayList;
+
+public class Board {
+
+    private final ArrayList<Pawn> pawns;
```

Comment:
다른 분들께도 같은 내용으로 안내해 드리고 있는데요, 
`ArrayList` 에만 있는 기능을 사용하실 게 아니라면, `List` 와 같이 타입을 지정해주시는 것이 좋습니다. 
`ArrayList` 에만 있는 기능을 사용한다 하더라도, 사용할 메소드에서 캐스팅해주면 되는데요, 
아직 이렇게 하는 이유가 공감이 되지 않을 수 있습니다. 이것은 나중에 차차 학습해 가도록 하고요, 학습 초반부에 잘못 들면 나중에 고치기 어려운 습관인지라 미리 리뷰를 드려요.

관련한 내용은 아래 링크에서 좀 더 자세히 읽어보세요. 
https://stackoverflow.com/questions/383947/what-does-it-mean-to-program-to-an-interface

```suggestion
    private final List<Pawn> pawns;

Reply:
해당 내용 찾아봤습니다. 읽어봐도 말씀대로 아직 크게 와닿지는 않네요ㅠㅠ 일단 보고 알게 된 것이 인터페이스를 사용하면 추후 다른 메서드(ArrayList에서만 사용가능한)가 꼭 필요한지 알 수 있고 그때 변경하더라도 타입만을 바꿔주면 되니 리팩토링도 어렵지 않다는 점이였습니다. 또한 메서드의 매개변수와 반환값에 인터페이스를 사용을하면 추후 변경 사항이 있더라도 호출 코드는 변경되지 않는다는 점을 알게되었습니다. 그 외에는 의식적으로 인터페이스를 사용하는 습관을 들여가면서 느껴보도록 하겠습니다!!

소중한 피드백 감사합니다!!